mgmt/ProxyConfig.h

```diff
@@ -117,15 +117,31 @@ ConfigScheduleUpdate(Ptr<ProxyMutex> &mutex)
   return 0;
 }
 
+template <typename UpdateClass>
```

Comment:
I don't understand this class. Why not just call `schedule_imm` directly at the place this is used?